docs/src/modules/java/pages/samples.adoc

```diff
@@ -17,6 +17,7 @@ Samples are available that demonstrate important patterns and abstractions. Thes
 | A customer registry with query capabilities | link:../java/_attachments/customer-registry-quickstart.zip[customer-registry-quickstart.zip] |Intermediate
 | A funds transfer workflow between two wallets | link:../java/_attachments/workflow-quickstart.zip[workflow-quickstart.zip] |Intermediate
 | A user registration service implemented as a Choreography Saga | link:../java/_attachments/choreography-saga-quickstart.zip[choreography-saga-quickstart.zip] |Advanced
+| Akka Chess: a complete, resilient, automatically scalable, event-sourced chess game | https://github.com/akka-samples/akka-chess[Github Repository] |Advanced
```

Comment:
Would be nice to migrate the UI part to plain HTML + JSish in this sample.

Reply:
Good suggestion, but I would argue that is outside the scope of this PR. I created an issue to track your idea https://github.com/lightbend/kalix/issues/12752.